src/vs/workbench/contrib/files/browser/fileActions.ts

```diff
@@ -251,7 +251,7 @@ function getMoveToTrashMessage(distinctElements: ExplorerItem[]): { message: str
 	}
 
 	if (distinctElements[0].isDirectory) {
-		return { message: nls.localize('confirmMoveTrashMessageFolder', "Are you sure you want to delete '{0}' and its contents?", distinctElements[0].name), detail: '' };
+		return { message: nls.localize('confirmMoveTrashMessageFolder', "Are you sure you want to delete '{0}' and its contents?", distinctElements[0].name), detail: '(You will not be able to restore it after exiting VS Code)' };
```

Comment:
This seems to be a "move to trash" message, in which case it's not true that you won't be able to restore it (the folder) after exiting VS Code

Reply:
Oh sorry that was a mistake, I didn't notice the `confirmMoveTrashMessageFolder` there.